### ISSUE TITLE
`Ember.String` deprecation warning as of Ember 4.10

### ISSUE THREAD

**Issue by jkantner** [DazzlingFugu/ember-cli-embedded#250]
I installed `ember-cli-embedded` after installing Ember 4.10, and I’m starting to see this deprecation warning in the console regards `Ember.String`:

```
DEPRECATION: Using `Ember.String` is deprecated. Please import methods directly from `@ember/string`. [deprecation id: ember-string.from-ember-module] This will be removed in ember-source 5.0.0. See https://deprecations.emberjs.com/v4.x/#toc_ember-string-from-ember-module for more details.
    @http://localhost:4200/assets/vendor.js:23477:17
    raiseOnDeprecation@http://localhost:4200/assets/vendor.js:23381:13
    @http://localhost:4200/assets/vendor.js:23477:17
    handleDeprecationWorkflow@http://localhost:4200/assets/vendor.js:56307:13
    @http://localhost:4200/assets/vendor.js:23477:17
    deprecationCollector@http://localhost:4200/assets/vendor.js:56342:9
    @http://localhost:4200/assets/vendor.js:23477:17
    @http://localhost:4200/assets/test-support.js:6076:15
    @http://localhost:4200/assets/vendor.js:23477:17
    invoke@http://localhost:4200/assets/vendor.js:23486:23
    deprecate@http://localhost:4200/assets/vendor.js:23454:28
    deprecateStringUseOnEmberModule@http://localhost:4200/assets/vendor.js:51126:51
    get@http://localhost:4200/assets/vendor.js:51141:38
    initialize@http://localhost:4200/assets/redacted.js:35895:36
    @http://localhost:4200/assets/vendor.js:24028:31
    @http://localhost:4200/assets/vendor.js:49389:11
    @http://localhost:4200/assets/vendor.js:49318:16
    @http://localhost:4200/assets/vendor.js:49263:26
    @http://localhost:4200/assets/vendor.js:49269:16
    _runInitializer@http://localhost:4200/assets/vendor.js:24052:20
    runInitializers@http://localhost:4200/assets/vendor.js:24026:27
    _bootSync@http://localhost:4200/assets/vendor.js:20514:29
    boot@http://localhost:4200/assets/vendor.js:20484:23
    @http://localhost:4200/assets/test-support.js:6362:32
    promiseReactionJob@[native code]
```

I haven’t used `String` anywhere in my app, and therefore I conclude this package is to blame.

**Comment by BlueCutOfficial** [comment]
Hey, sorry for not answering sooner, we are just a few people maintaining this when we have a bit of time. Is the issue still open or did you find a different root cause in the end?

**Comment by MrChocolatine** [comment]
The add-on has been upgraded to the latest version available (see #305).
If you can upgrade, let us know if you still encounter your original issue, if you don't then this issue can be closed.

**Comment by MrChocolatine** [comment]
@jkantner 

The add-on has been upgraded to the latest version available (see #305).
If you can upgrade, let us know if you still encounter your original issue, if you don't then this issue can be closed.

**Comment by MrChocolatine** [comment]
Closing this ticket due to inactivity.
